### PR TITLE
OK - all tests passing -

### PR DESCRIPTION
**Pull request by yamlcamel**

### CODE REVIEW

File: model/booking_handler.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class Booking_handler
```
Comment (jjegarcia) [inline model/booking_handler.rb]: can you please rename class to BookingHandler?

**Review by JRMBowers — APPROVED** [review]

File: model/database.rb [hunk 2]
```diff
@@ -1,4 +1,3 @@
-
 class Database
 
   def self.connect(database)
```
Comment (jjegarcia) [inline model/database.rb:3]: please delete empty function

File: model/space_handler.rb [hunk 3]
```diff
@@ -22,5 +22,4 @@ def self.get_spaces
     result = Database.query('SELECT * FROM spaces;')
```
Comment (jjegarcia) [inline model/space_handler.rb:22]: have you run rubocop -A. This very long line may require splitting

File: spec/booking_handler_spec.rb [hunk 4]
```diff
@@ -0,0 +1,40 @@
+require_relative '../model/booking_handler'
+require_relative '../model/space_handler'
+
+describe Booking_handler do
+  describe '.add_booking' do
+    it 'add_bookings a add booking in the database' do
+      populate_spaces_table
+      #      persist_space_id(table: 'spaces')
```
Comment (jjegarcia) [inline spec/booking_handler_spec.rb:8]: please remove commented out lines

File: spec/booking_handler_spec.rb [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      bookings = Booking_handler.get_bookings
+    
+      expect(bookings.length).to eq 3
+      expect(bookings.first).to be_a Booking_handler
+      expect(bookings.first.space_id).to eq booking.space_id
+    end
+  end
+
+  describe '.available' do
+ #   it 'confirms whether the property is available' do
```
Comment (jjegarcia) [inline spec/booking_handler_spec.rb:40]: please delete commented out lines

**Review by jjegarcia — CHANGES_REQUESTED** [review]

**Pull request closed without merging**